controllers/articles-controller.js

```diff
@@ -1,9 +1,24 @@
-const { selectAllArticles } = require("../models/articles-model.js");
+const { selectAllArticles, selectArticleById } = require("../models/articles-model.js");
 
 exports.getAllArticles = (req, res, next) => {
     selectAllArticles()
     .then((allArticles) => {
         res.status(200).send({allArticles: allArticles})
     })
     .catch(next)
+};
+
+exports.getArticleById = (req, res, next) => {
+    const id = req.params.article_id; 
+    const isValidInput = (/^\d*$/);

```

Comment:
Nice logic, but PSQL will automatically create this error for you when you try to put in an incorrect data type.